cpptod.dd

```diff
@@ -829,6 +830,84 @@ void test()
 }
 ------
 
+<hr>$(COMMENT  -------------------------------------------- )
+
+$(H3 <a name="interfaces">Interfaces</a>)
+
+Interfaces are used to describe the behavior or capabilities of a class without
+commiting to a particular implementation of the class.
+
+$(H4 The C++ Way)
+
+  In C++ an interface is implemented with an `Abstract Base Class`. The ABC defines
```

Comment:
I believe in C++ they are just referred to as "abstract classes" without the "base".  I suggest  "In C++, an interface can be implemented using an abstract class.  The abstract class  defines..."